cmd/crossplane/core/core.go

```diff
@@ -66,6 +66,7 @@ type startCommand struct {
 	LeaderElection       bool   `short:"l" help:"Use leader election for the controller manager." default:"false" env:"LEADER_ELECTION"`
 	Registry             string `short:"r" help:"Default registry used to fetch packages when not specified in tag." default:"${default_registry}" env:"REGISTRY"`
 	CABundlePath         string `help:"Additional CA bundle to use when fetching packages from registry." env:"CA_BUNDLE_PATH"`
+	ValidationPem        string `help:"Private key in PEM format to be used for package signature validation." env:"VALIDATION_PEM"`
```

Comment:
`CABundlePath` is a somewhat similar scenario as the signature verification. In that case we went with supplying a path rather than the literal contents of the cert. I think that could also make sense in this use-case to allow mounting PEM-encoded pub key from a variety of sources (e.g. `Secret`).